tests/provenance/path-records/path.dl

```diff
@@ -0,0 +1,19 @@
+// Souffle - A Datalog Compiler
```

Comment:
It should be path-records/path-records.dl, and it should also be referenced in provenance.at